packages/styles/src/core/layout/toolbar/_tabs.scss

```diff
@@ -1,37 +1,27 @@
+$_toolbar-height: 13 * $u1;
```

Comment:
Two things:
- If we need `13 * $u1`, let's make `$13u1`.
- Are we using `$toolbar-height` also in widths? When the toolbar is vertical.

Reply:
I don't see that `$13u1`, honestly, because we would need to create many variables that might be used once.

And personally, I understand better what `13 * $u1` is, rather than using `$13u1`.

What I don't really like is the `$u1`, but that's a naming thing.